braid/twisted/service.py

```diff
@@ -54,10 +54,12 @@ def bootstrap(self, python='pypy'):
             pip.install('twisted', python=python)
 
             # Create base directory setup
-            run('/bin/mkdir -p {} {} {}'.format(
+            run('/bin/mkdir -p {} {} {} {}'.format(
                 self.runDir,
                 self.logDir,
-                self.binDir))
+                self.binDir,
+                self.configDir,
+                ))
```

Comment:
There are 4 `{}`s here and 5 parameters to `.format`, which seems like a problem.